src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2302,6 +2302,37 @@ The `code` property accepts human-readable versions of all HTTP status codes. Th
 
 The `message` property accepts a string. (e.g. "User must be logged in.")
 
+## Environment variables (`astro:env`)
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `astro:env` API lets you configure a type-safe schema for environment variables you have set. This allows you to indicate whether they should be available on the server or the client, and define their data type and additional properties. For examples and usage instructions, [see the `astro:env` guide](/en/guides/environment-variables/#type-safe-environment-variables).
+
+### `getSecret()`
+
+<p>
+<Since v="5.0.0" />
+</p>
+
+The `getSecret()` helper function allows retrieving the raw value of an environment variable by its key. It has 2 usecases:
+
+1. **Get the raw value of a secret defined in your schema**: if you define a variable `FOO` of type boolean, using `getSecret()` will allow you to get the value as string:
+
+   ```js
+   import {
+      FOO, // boolean
+      getSecret
+   } from 'astro:env/server'
+
+   getSecret('FOO') // string | undefined
+   ```
+
+2. **Retrieve secrets not specified in your schema**: that can be useful if you need to get a secret depending on dynamic data, eg. a database or API.
+
+If you need to retrieve environment variables programmatically, we recommend you use `getSecret()` instead of `process.env` (or equivalent) because its implementation is provided by your adapter: It means you won't need to update all your calls if you switch adapters. It defaults to `process.env` in dev and build.
```

Comment:
```suggestion
The `getSecret()` helper function allows retrieving the raw value of an environment variable by its key.

For example, you can retrieve a boolean value as a string:

   ```js
   import {
      API_KEY, // boolean
      getSecret
   } from 'astro:env/server'

   getSecret('API_KEY') // string | undefined
   ```

This can also be useful to get a secret depending on dynamic data, such as from a database or API.

If you need to retrieve environment variables programmatically, we recommend using `getSecret()` instead of `process.env` (or equivalent). Because its implementation is provided by your adapter, you won't need to update all your calls if you switch adapters.  It defaults to `process.env` in dev and build.
```